fe/fe-core/src/main/java/com/starrocks/connector/iceberg/hive/IcebergHiveCatalog.java

```diff
@@ -66,8 +67,10 @@ public class IcebergHiveCatalog implements IcebergCatalog {
     @VisibleForTesting
     public IcebergHiveCatalog(String name, Configuration conf, Map<String, String> properties) {
         this.conf = conf;
-        this.conf.set(MetastoreConf.ConfVars.CLIENT_SOCKET_TIMEOUT.getHiveName(),
-                String.valueOf(Config.hive_meta_store_timeout_s));
```

Comment:
why not use this configuration?

Reply:
If user does not set the new added propertiy `hive.metastore.timeout` when creating catalog, then will use this default value `Config.hive_meta_store_timeout_s`.